MonsterTrainModdingAPI/Patches/LoadCustomArtPatch.cs

```diff
@@ -23,7 +23,11 @@ static void Prefix(ref CardData __instance)
         {
             if (!__instance.HasCardArtPrefabVariant() && CustomCardManager.CustomCardData.ContainsKey(__instance.GetID()))
             {
-                CustomCardManager.CreateCardGameObject(__instance.GetID());
+                GameObject obj = CustomCardManager.CreateCardGameObject(__instance.GetID());
+                if (obj != null)
+                {
+                    UnityEngine.Object.DontDestroyOnLoad(obj);
```

Comment:
Are we sure about this? Seems like it could result in extremely high memory usage.

Reply:
Unity does this in the background for prefab game objects.